logs/AuthenticationService.log

```diff
@@ -0,0 +1,14 @@
+2021-12-05 18:01:39,845 INFO mate.academy.service.AuthenticationServiceImpl:13 login method was called!. Params: login=bob
+2021-12-05 18:01:39,883 INFO mate.academy.service.AuthenticationServiceImpl:15 method completeOrder was called. Params: userId=2
+2021-12-05 18:01:39,899 INFO mate.academy.service.AuthenticationServiceImpl:31 successful fetched data from DB. Params: userId=2
+2021-12-05 18:05:40,977 INFO mate.academy.service.AuthenticationServiceImpl:13 login method was called!. Params: login=bob
+2021-12-05 18:05:40,993 INFO mate.academy.service.AuthenticationServiceImpl:15 method completeOrder was called. Params: userId=2
+2021-12-05 18:05:40,993 INFO mate.academy.service.AuthenticationServiceImpl:31 successful fetched data from DB. Params: userId=2
+2021-12-05 18:06:08,040 INFO mate.academy.service.AuthenticationServiceImpl:13 login method was called!. Params: login=bob
+2021-12-05 18:06:08,040 ERROR mate.academy.service.AuthenticationServiceImpl:21 Can't login
+mate.academy.exception.AuthenticationException: Username or password are incorrect
+	at mate.academy.service.AuthenticationServiceImpl.login(AuthenticationServiceImpl.java:16) ~[classes/:?]
+	at mate.academy.Main.main(Main.java:19) [classes/:?]
+2021-12-05 18:18:14,843 INFO mate.academy.service.AuthenticationServiceImpl:13 login method was called!. Params: login=bob
+2021-12-05 18:18:14,850 INFO mate.academy.service.AuthenticationServiceImpl:15 method completeOrder was called. Params: userId=2
+2021-12-05 18:18:14,850 INFO mate.academy.service.AuthenticationServiceImpl:31 successful fetched data from DB. Params: userId=2
```

Comment:
"Edit .gitignore file so that your file with logs is NOT pushed to Github" lets remove all log files from PR